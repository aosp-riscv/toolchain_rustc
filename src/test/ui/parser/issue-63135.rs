<<<<<<< HEAD   (086005 Importing rustc-1.38.0)
// error-pattern: aborting due to 6 previous errors
=======
// error-pattern: aborting due to 5 previous errors
>>>>>>> BRANCH (8cd2c9 Importing rustc-1.39.0)

fn i(n{...,f #
