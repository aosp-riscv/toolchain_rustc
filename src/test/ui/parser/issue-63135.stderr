error: this file contains an un-closed delimiter
  --> $DIR/issue-63135.rs:3:16
   |
LL | fn i(n{...,f #
   |     - -        ^
   |     | |
   |     | un-closed delimiter
   |     un-closed delimiter

error: expected field pattern, found `...`
  --> $DIR/issue-63135.rs:3:8
   |
LL | fn i(n{...,f #
   |        ^^^ help: to omit remaining fields, use one fewer `.`: `..`

error: expected `}`, found `,`
  --> $DIR/issue-63135.rs:3:11
   |
LL | fn i(n{...,f #
   |        ---^
   |        |  |
   |        |  expected `}`
   |        `..` must be at the end and cannot have a trailing comma

error: expected `[`, found `}`
  --> $DIR/issue-63135.rs:3:15
   |
LL | fn i(n{...,f #
   |               ^ expected `[`

<<<<<<< HEAD   (086005 Importing rustc-1.38.0)
error: expected `:`, found `)`
  --> $DIR/issue-63135.rs:3:15
   |
LL | fn i(n{...,f #
   |               ^ expected `:`

error: expected one of `->`, `where`, or `{`, found `<eof>`
  --> $DIR/issue-63135.rs:3:15
   |
LL | fn i(n{...,f #
   |               ^ expected one of `->`, `where`, or `{` here

error: aborting due to 6 previous errors
=======
error: expected one of `:` or `|`, found `)`
  --> $DIR/issue-63135.rs:3:15
   |
LL | fn i(n{...,f #
   |               ^ expected one of `:` or `|` here

error: aborting due to 5 previous errors
>>>>>>> BRANCH (8cd2c9 Importing rustc-1.39.0)

